pkg/controller/virtinformers.go

```diff
@@ -892,6 +895,15 @@ func (f *kubeInformerFactory) ExportRouteConfigMap() cache.SharedIndexInformer {
 	})
 }
 
+func (f *kubeInformerFactory) CpuModelToFeaturesToDisableConfigMap() cache.SharedIndexInformer {
+	return f.getInformer("cpuModelToFeaturesToDisable", func() cache.SharedIndexInformer {
+		restClient := f.clientSet.CoreV1().RESTClient()
+		fieldSelector := fields.OneTermEqualSelector("metadata.name", "cpu-model-to-features-to-disable")
```

Comment:
Well i think we better keep this hard-coded because:
- Similar cases are already hard coded:

https://github.com/kubevirt/kubevirt/blob/06c255965a28e9c254140f18d8c06eaa8962c6fa/pkg/controller/virtinformers.go#L889
https://github.com/kubevirt/kubevirt/blob/06c255965a28e9c254140f18d8c06eaa8962c6fa/pkg/controller/virtinformers.go#L880

So we should keep things consistent(it can be changed all together in a future PR).

- The ConfigMap name shouldn't be a global variable  because global vars are [anti pattern](https://wiki.c2.com/?GlobalVariablesAreBad) 

- Because the ConfigMap name appear only once in the project(not including tests).

Reply:
Done.